File: SDK-v5/localytics-with-gcm/app/src/main/java/com/localytics/android/sample/gcm/MainActivity.java
package com.localytics.android.sample.gcm;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.localytics.android.Localytics;

public class MainActivity extends AppCompatActivity
{

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Localytics.registerPush("YOUR_SENDER_ID");
    }

    @Override
    protected void onNewIntent(Intent intent)
    {
        super.onNewIntent(intent);

        Localytics.onNewIntent(this, intent);
    }
}

File: SDK-v5/kotlin/localytics-with-gcm/app/src/main/java/com/localytics/android/sample/gcm/MainActivity.kt
package com.localytics.android.sample.gcm

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.localytics.android.Localytics

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Localytics.registerPush()
    }

    override fun onNewIntent(intent: Intent) {
        super.onNewIntent(intent)

        Localytics.onNewIntent(this, intent)
    }
}
